### ISSUE TITLE
Chef-client run will fail on bootstrap

### ISSUE THREAD

**Issue by priestjim** [chef-cookbooks/chef-splunk#26]
While bootstrapping a node with the chef-splunk recipe on the default runlist I get this error:

```
NoMethodError: undefined method `split' for #<Hash:0x007fdd20381c10>
/var/cache/chef/cookbooks/chef-splunk/recipes/setup_auth.rb:23:in `from_file'
```

I need to update the vault after the node has been registered (which means after the crash) in order for that part to succeed.

**Comment by vidkun** [comment]
I'm seeing the same issue. First run during bootstrap gets an error that the vault item is not encrypted with this node's key. If I update the vault item to include the new node, the next run gives the error above about "undefined method 'split'...."

Happy to help troubleshoot further if needed.

**Comment by bby-bishopclark** [comment]
#72 would solve this issue as well, I suspect.

**Comment by haidangwa** [comment]
closing this old issue. Chef 13.4 included the chef-vault gem and v2.0.0 of this cookbook required `chef_version '>= 13.11'`, as well. If it is still a problem, please confirm with a modern version of chef-client and open a new issue.
